pycparser/c_parser.py

```diff
@@ -310,6 +310,11 @@ def _fix_decl_name_type(self, decl, typename):
         decl.name = type.declname
         type.quals = decl.quals[:]
 
+        for tn in typename:
+            if isinstance(tn, c_ast.Alignas):
+                type.align = tn
```

Comment:
I'm a bit confused here: isn't align an attribute, not a child node? Should it be a child node instead if you're assigning nodes to it?

Reply:
It is an attribute indeed, not a child node. This code was merely a fixup for the parser limitation. Previously one of the tests triggered an issue without this code, however, currently none does. I am in a good faith to believe that the issue was resolved more properly in one of the future iterations of the code and I did not notice it. Will simply remove.